tdms/include/simulation_parameters.h

```diff
@@ -32,6 +33,17 @@ enum InterpolationMethod{
   band_limited
 };
 
+/**
+ * A three-tuple of integers that contain the stride in each direction
+ * to extract the phasors on. the surface i.e. x = 2 means extract from
+ * every 2nd Yee cell.
+ */
+struct PhasorInc{
```

Comment:
Why `Inc` ?

Reply:
once again: not a clue. in the absence of a better name I left it what it was